synapse/config/registration.py

```diff
@@ -323,6 +324,11 @@ def generate_config_section(self, generate_secrets=False, **kwargs):
         # users cannot be auto-joined since they do not exist.
         #
         #autocreate_auto_join_rooms: true
+
+        # When auto_join_rooms is specified, setting this flag to False prevents
+        # guest accounts from being automatically joined to the rooms. Default
+        # true when auto_join_rooms is specified.
```

Comment:
please please please follow the conventions. https://github.com/matrix-org/synapse/blob/develop/docs/code_style.md#configuration-file-format

```suggestion
        # true when auto_join_rooms is specified.
        #
```

Reply:
I have to admit, after trying to appease the linter so much I thought everything was covered.

Reply:
congratulations, you managed to find something we haven't covered with the linter.

you'll need to regenerate the sample config now.